moveit_experimental/moveit_jog_arm/CMakeLists.txt

```diff
@@ -10,12 +10,21 @@ endif()
 find_package(catkin REQUIRED COMPONENTS
   control_msgs
   geometry_msgs
+  message_generation
   moveit_ros_planning_interface
   rosparam_shortcuts
   tf2_ros
 )
 find_package(Eigen3 REQUIRED)
 
+add_service_files(
+  FILES
+  ChangeControlDimensions.srv
```

Comment:
I think this should go in the service folder of moveit_msgs... to date the /moveit repo does not have a single message or service type.

(I also think moveit_msgs should be in the /moveit repo, but that's a different discussion)